keycloak/sms-provider/src/main/java/org/sunbird/sms/MessageProviderFactory.java

```diff
@@ -0,0 +1,31 @@
+package org.sunbird.sms;
```

Comment:
We should avoid this file.. can't you dynamically create the factory classname in KeycloakSmsAuthenticatorUtil.sendSmsCode() 

cc @vrayulu @kochhar

Reply:
@coolbung Its just like a factory class where singleton object creation happens for all providers at one place

Reply:
@shriharshs things seem to be getting a bit complicated. let's review this design today.